src/app-layout/visual-refresh/trigger-button.tsx

```diff
@@ -103,6 +227,14 @@ function TriggerButton(
           {badge && <div className={styles.dot} />}
         </>
       )}
+      {tooltipVisible && (
+        <Tooltip
+          trackRef={containerRef}
+          value={tooltipValue}
+          className={styles['trigger-tooltip']}
+          {...(!isMobile && { position: 'left' })}
```

Comment:
Is `position: left` necessary? The tooltip uses the popover internally, which manages its position automatically according to the space available.

Reply:
yes. for UI consistency so all tooltips show in a consistent way